### PR TITLE
Annotations

### PR DESCRIPTION
**Pull request by Vincz**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documented?   | yes
| Fixed tickets | #349 
| License       | MIT

The annotations features.

### CODE REVIEW

**Comment by mcg-web** [comment]
Thank you for this PR @Vincz! To be ready to merge, you should fix phpcs using `composer fix-cs`. Also lint the project to fix some untested part of the code with `composer static-analysis`.

File: src/ExpressionLanguage/ExpressionFunction/GraphQL/ValueResolver.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Overblog\GraphQLBundle\ExpressionLanguage\ExpressionFunction\GraphQL;
+
+use Overblog\GraphQLBundle\ExpressionLanguage\ExpressionFunction;
+
+final class ValueResolver extends ExpressionFunction
```
[the remark this answers is not in the document]
Comment (Vincz) [reply src/ExpressionLanguage/ExpressionFunction/GraphQL/ValueResolver.php]: You're right. It was useless and I removed it.

**Comment by Vincz** [comment]
It should be ok now :)

File: docs/definitions/expression-language.md [hunk 2]
```diff
@@ -5,38 +5,39 @@ All definition config entries can use expression language but it must be explici
...
+| mixed **parameter**(string $name)                                      | Get parameter from the container                                                                                   | @=parameter('kernel.debug')                        | param |
+| boolean **isTypeOf**(string $className)                                | Verified if `value` is instance of className                                                                       | @=isTypeOf('AppBundle\\User\\User')                |
+| mixed **resolver**(string $alias, array $args = [])                    | call the method on the tagged service "overblog_graphql.resolver" with args                                        | @=resolver('blog_by_id', [value['blogID']]         | res   |
+| mixed **value_resolver**(array $args = [], string $method = null)      | call a method on the value itself. If no method name is provided, default to resolving fieldname (info->fieldName) | @=value_resolver([args["gender"]], "getFriends")   |
```
Comment (mcg-web) [inline docs/definitions/expression-language.md]: you should also removed this

Comment (Vincz) [reply docs/definitions/expression-language.md]: Done :)

**Comment by mcg-web** [comment]
Thank you @Vincz :+1:

**Pull request merged**